cmake/external/eigen.cmake

```diff
@@ -12,6 +12,13 @@ if(NOT WITH_FAST_MATH)
   add_definitions(-DEIGEN_FAST_MATH=0)
 endif()
 
+if(WIN32)
+    set(PATCH_COMMAND ${CMAKE_COMMAND} -E copy_if_different
+            ${PADDLE_SOURCE_DIR}/patches/eigen/Half.h.txt
+            ${EIGEN_INCLUDE_DIR}/Eigen/src/Core/arch/CUDA/Half.h)
```

Comment:
**I don't think we should do that for the users who need to compile Paddle in Windows, we should let users do that themselves to modify this file.**

Reply:
remove the patch